### ISSUE TITLE
Not working with socket.io module

### ISSUE THREAD

**Issue by paulking00** [niutech/node.php#12]
Hi niutech,

I really love this script. Its working great for node apps, but doesnt seem to be connecting properly for socket.io connections

I've tried implementing even the most basic demo from http://socket.io/demos/chat/ but I cant seem to get it to connect. 

Maybe I am missing something. Have you successfully run a socket.io demo on your node.php installation?

Many thanks for a great script! Would be awesome to have the powerful socket.io module working :-)

**Comment by niutech** [comment]
Thank you!

I have installed a [socket.io chat demo](http://mobiliot.com/node.php) using node.php. The only things I had to modify were paths (`node.php?path=style.css` instead of `style.css` in `public/index.html`, etc.) and a rewrite rule from `/socket.io/` to `node.php?path=socket.io/` in `.htaccess`.

However, socket.io client cannot establish a connection with socket.io server using websocket (because node.php does not support it) and long polling does not work as intended (because cURL waits until a server closes connection before it sends back a response to a client).

For now, I do not have any good solution for that.

**Comment by niutech** [comment]
Thank you!

I have installed a [socket.io chat demo](http://mobiliot.com/node.php) using node.php. The only things I had to modify were paths (`node.php?path=style.css` instead of `style.css` in `public/index.html`, etc.) and rewrite rule from `/socket.io/` to `node.php?path=socket.io/` in `.htacess`.

However, socket.io client cannot establish a connection with socket.io server using websocket (because node.php does not support it) and long polling does not work as intended (because cURL waits until a server closes connection before it sends back a response to a client).

For now, I do not have any good solution.
